src/components/DashboardHome/Sidebar.jsx

```diff
@@ -2,10 +2,27 @@ import React from 'react';
 
 import Links from '../Links/Links';
 import linksData from './sampleLinks';
+import SidebarBlock from './SidebarBlock';
 
 const Sidebar = () => (
   <>
-    <Links id={linksData.id} title={linksData.title} links={linksData.links} />
+    <SidebarBlock title="Program Document" sectionClassName="links">
+      <Links id={linksData.id} links={linksData.links} />
+    </SidebarBlock>
+    <SidebarBlock title="Manage Your Degree">
+      <p>This could be any bunch of children passed in somehow.</p>
```

Comment:
Just to match what's in the mocks, let's have the children of this `SidebarBlock` say:

![image](https://user-images.githubusercontent.com/2828721/58127103-938fa200-7be2-11e9-9a9c-be4263270cf6.png)

including the external link at the bottom with the appropriate `FontAwesomeIcon` (https://fontawesome.com/icons/external-link-alt?style=solid).

Reply:
Note: For external links, you'll need to include the following attributes on the `<a>` element: `target="_blank" rel="noopener noreferrer"`

Reply:
Will do... though, `an advisor` :)